h2o-py/h2o/model/metrics_base.py

```diff
@@ -1766,6 +1768,55 @@ def gains_lift(self):
             return self._metric_json['gains_lift_table']
         return None
 
+    @deprecated_params({'save_to_file': 'save_plot_path'})
+    def plot_gains_lift(self, type="both", server=False, save_plot_path=None, plot=True, **kwargs):
```

Comment:
guess what…
I wouldn't mind if:
1. `kwargs` was then passed to matplotlib somehow to customize plots: shouldn't be necessary though now that we return the figure.
2. it was documented.